src/main/resources/db/migration/V1.41__Add_Claim_Event_Filtering_To_Beacon_Search_View.sql

```diff
@@ -0,0 +1,32 @@
+CREATE OR REPLACE VIEW beacon_search AS
+SELECT id,
+       created_date,
+       last_modified_date,
+       (CASE
+            WHEN (SELECT COUNT(*) FROM claim_event WHERE claim_event.legacy_beacon_id = legacy_beacon.id) = 0
+                THEN 'MIGRATED'
+            ELSE 'DELETED' END) AS beacon_status,
+       hex_id,
+       owner_name,
+       owner_email,
+       NULL                     as account_holder_id,
+       use_activities
+FROM legacy_beacon
+UNION
+SELECT id,
+       created_date,
+       last_modified_date,
+       beacon_status,
+       hex_id,
+       (SELECT full_name FROM person WHERE person.person_type = 'OWNER' AND person.beacon_id = beacon.id) AS owner_name,
+       (SELECT email
+        FROM person
+        WHERE person.person_type = 'OWNER'
+          AND person.beacon_id = beacon.id)                                                               AS owner_email,
+       (SELECT id
+        FROM account_holder
+        WHERE account_holder.id = beacon.account_holder_id)                                               AS account_holder_id,
+       (SELECT REPLACE(string_agg(activity, ', '), '_', ' ')
+        FROM beacon_use
+        WHERE beacon_use.beacon_id = beacon.id)                                                           AS use_activities
+FROM beacon;
```

Comment:
Need a `'legacy_beacon' AS beacon_type` and `'beacon' AS beacon_type` on the view